openff/evaluator/backends/dask.py

```diff
@@ -101,6 +101,11 @@ def _wrapper(func, queue, args, kwargs):
                             worker_logger.info(
                                 f"LSBJOBID: {os.environ.get('LSB_JOBID')}"
                             )
+                        elif os.environ.get("SLURM_JOBID") is not None:
```

Comment:
```suggestion
                        elif os.environ.get("SLURM_JOB_ID") is not None:
```
I think `SLURM_JOB_ID` is the newer variable name?

Reply:
I have both in the env variables on my cluster but slurm manual suggests the newer one